storage/rocksdb/rdb_datadic.cc

```diff
@@ -4303,14 +4277,17 @@ bool Rdb_ddl_manager::rename(const std::string &from, const std::string &to,
 }
 
 void Rdb_ddl_manager::cleanup() {
-  my_hash_free(&m_ddl_hash);
+  for (auto it = m_ddl_hash.cbegin(); it != m_ddl_hash.cend(); it++)
```

Comment:
const auto & it, also consider switching to range-based loop const auto & it : m_ddl_hash